packages/ag-charts-community/src/chart/series/cartesian/lineSeries.ts

```diff
@@ -156,7 +156,8 @@ export class LineSeries extends CartesianSeries<LineContext> {
         this.dataModel = dataModel;
         this.processedData = processedData;
 
-        if (processedData.reduced?.diff?.added.length > 0 && processedData.reduced?.diff?.removed.length > 0) {
+        // If the diff is too complex then just clear and redraw to prevent wonky line wobbling
+        if (processedData.reduced?.diff?.added.length > 1 && processedData.reduced?.diff?.removed.length > 1) {
```

Comment:
This is definitely a little hacky, but we need to cover two common use cases:

- Many points are changing -> do a clear and full redraw
- The first point is removed and a new point added onto the end -> animate normally (see the animation in the main PR description)

The other way to differentiate these would require a bunch of pre-processing of the data and diff before calling the transition. There may be a nicer way to do that, but I'll look at that when I do a greater refactor of the line animation out of the current big blob of logic.